ts-obs/cmd/chunkIntervalReset.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/spf13/cobra"
+)
+
+// chunkIntervalResetCmd represents the chunk-interval reset command
+var chunkIntervalResetCmd = &cobra.Command{
+	Use:   "reset <metric>",
+	Short: "Resets the chunk interval for a specific metric back to the default",
+	Args:  cobra.ExactArgs(1),
+	RunE:  chunkIntervalReset,
+}
+
+func init() {
+	chunkIntervalCmd.AddCommand(chunkIntervalResetCmd)
+	chunkIntervalResetCmd.Flags().StringP("user", "U", "postgres", "database user name")
+	chunkIntervalResetCmd.Flags().StringP("dbname", "d", "postgres", "database name to connect to")
+}
+
+func chunkIntervalReset(cmd *cobra.Command, args []string) error {
+	var err error
+
+	metric := args[0]
+
+	var user string
+	user, err = cmd.Flags().GetString("user")
+	if err != nil {
+		return fmt.Errorf("could not reset chunk interval for %v: %w", metric, err)
+	}
+
+	var dbname string
+	dbname, err = cmd.Flags().GetString("dbname")
+	if err != nil {
+		return fmt.Errorf("could not reset chunk interval for %v: %w", metric, err)
+	}
+
+	var name string
+	name, err = cmd.Flags().GetString("name")
+	if err != nil {
+		return fmt.Errorf("could not reset chunk interval for %v: %w", metric, err)
+	}
+
+	var namespace string
+	namespace, err = cmd.Flags().GetString("namespace")
+	if err != nil {
+		return fmt.Errorf("could not reset chunk interval for %v: %w", metric, err)
+	}
+
+	secret, err := KubeGetSecret(namespace, name+"-timescaledb-passwords")
+	if err != nil {
+		return fmt.Errorf("could not get TimescaleDB password: %w", err)
+	}
+
+	pass := string(secret.Data[user])
+
+	podName, err := KubeGetPodName(namespace, map[string]string{"release": name, "role": "master"})
+	if err != nil {
+		return fmt.Errorf("could not reset chunk interval for %v: %w", metric, err)
+	}
+
+	err = KubePortForwardPod(namespace, podName, LISTEN_PORT_TSDB, FORWARD_PORT_TSDB)
```

Comment:
for these types of "internal" port forward the local port should be 0. and you should use https://godoc.org/k8s.io/client-go/tools/portforward#PortForwarder.GetPorts to get the local port actually used. The reason is LISTEN_PORT_TSDB may already be in use locally by something else, and so it's better to have the OS just pick a free port.